Libraries/libcar/Headers/car/car_format.h

```diff
@@ -196,6 +198,11 @@ struct car_header {
     uint32_t key_semantics; // 01 00 00 00
 } __attribute__((packed));
 
+struct car_extended_metadata {
+    char magic[4]; // 'META'
+    char contents[1024]; // string
```

Comment:
Hmm, I thought it was a fixed-length format since there was no size field, but I realize now that that could be computed from the BOM index size containing the extended metadata. I think fixed-size makes more sense, though...